coreum/chain.json

```diff
@@ -101,23 +101,23 @@
       },
       {
         "name": "v3",
-        "tag": "v3.0.2",
+        "tag": "v3.0.3",
         "proposal": 8,
         "height": 13480000,
-        "recommended_version": "v3.0.2",
+        "recommended_version": "v3.0.3",
         "compatible_versions": [
-          "v3.0.2"
+          "v3.0.3"
```

Comment:
is v3.0.2 compatible with v3.0.3?

Reply:
Yes, it's compatible it terms of the chain. The `v3.0.3` contains the security fixes.